{{cookiecutter.project_slug}}/config/settings/local.py

```diff
@@ -76,8 +76,10 @@
 
 # Celery
 # ------------------------------------------------------------------------------
+{% if cookiecutter.use_docker == 'n' -%}
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-always-eager
 CELERY_TASK_ALWAYS_EAGER = True
+{%- endif % }
```

Comment:
Just realised that there is another extra space causing a syntax error here:

```suggestion
{%- endif %}
```

Reply:
The rest looks good now.